hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/fees/calculation/crypto/queries/GetAccountInfoResourceUsage.java

```diff
@@ -82,4 +85,28 @@ public FeeData usageGiven(final Query query, final StateView view, final Map<Str
                 .build();
         return cryptoOpsUsage.cryptoInfoUsage(query, ctx);
     }
+
+    /**
+     * This method is used to calculate the fee for the {@code CryptoGetInfo}
+     * query in modularized code only.
+     * @param query query to be processed
+     * @param account account whose info is being retrieved
+     * @return the fee for the query
+     */
+    public FeeData usageGiven(final Query query, final Account account) {
```

Comment:
`query` is a protobuf-object while `account` is a pbj-object. I think you should make this consistent and use either protobuf or pbj. The same is true for the other new methods below.